server/ServerApp.cpp

```diff
@@ -101,17 +101,17 @@ ServerApp::ServerApp() :
     InfoLogger() << FreeOrionVersionString();
     LogDependencyVersions();
 
-    m_galaxy_setup_data.seed = GetOptionsDB().Get<std::string>("setup.seed");
-    m_galaxy_setup_data.size = GetOptionsDB().Get<int>("setup.star.count");
-    m_galaxy_setup_data.shape = GetOptionsDB().Get<Shape>("setup.galaxy.shape");
-    m_galaxy_setup_data.age = GetOptionsDB().Get<GalaxySetupOption>("setup.galaxy.age");
-    m_galaxy_setup_data.starlane_freq = GetOptionsDB().Get<GalaxySetupOption>("setup.starlane.frequency");
-    m_galaxy_setup_data.planet_density = GetOptionsDB().Get<GalaxySetupOption>("setup.planet.density");
-    m_galaxy_setup_data.specials_freq = GetOptionsDB().Get<GalaxySetupOption>("setup.specials.frequency");
-    m_galaxy_setup_data.monster_freq = GetOptionsDB().Get<GalaxySetupOption>("setup.monster.frequency");
-    m_galaxy_setup_data.native_freq = GetOptionsDB().Get<GalaxySetupOption>("setup.native.frequency");
-    m_galaxy_setup_data.ai_aggr = GetOptionsDB().Get<Aggression>("setup.ai.aggression");
-    m_galaxy_setup_data.game_uid = GetOptionsDB().Get<std::string>("setup.game.uid");
+    m_galaxy_setup_data.seed           = GetOptionsDB().Get<std::string>("setup.seed");
```

Comment:
I don't know if this is causing trouble with Black linter, but the other pieces of code before this PR had the "=" to the left of the padding.

Reply:
thanks, but since the only test that is still failing has as its name something with python formatting, i do not believe this is the cause of that.

<img width="865" alt="only python test still failing" src="https://user-images.githubusercontent.com/37298400/153721842-b68ddcc2-1c3d-4358-b56c-104fd8efe146.png">

Reply:
can try it anyway though

Reply:
Gah! I was aware it was python stuff but I didn't realize this was C++ :_(

Reply:
yep. i overlooked one of those. that was it. all in the green now.